docs/.vitepress/theme/composables/useSearchInput.ts

```diff
@@ -1,43 +1,40 @@
-import { refThrottled } from '@vueuse/core';
+import { useDebounce } from '@vueuse/core';
 import { nextTick, onMounted, ref, watch } from 'vue';
 
 const useSearchInput = () => {
-  const searchInput = ref()
+  const searchInput = ref();
   const searchQuery = ref(
     typeof window === 'undefined'
-    ? ''
-    : (
-      new URLSearchParams(window.location.search).get('search')
-      || ''
-      )
-  )
-  const searchQueryThrottled = refThrottled(searchQuery, 400)
+      ? ''
+      : new URLSearchParams(window.location.search).get('search') || ''
+  );
+  const searchQueryDebounced = useDebounce(searchQuery, 500);
```

Comment:
After testing this, I feel like it's a bit too slow for a search field.
```suggestion
  const searchQueryDebounced = useDebounce(searchQuery, 250);
```